### DISCUSSION TITLE
Replicated Montage

### DISCUSSION THREAD

**Discussion by eclipsisme** [Sixze/ALS-Refactored#142] [General]
I couldn't find anything while looking through the code. Is there any built in support for playing a replicated Montage?

**Comment by falbellaihi1** [comment]
What do you mean by replicated montage? if you want to play a montage that shows to other clients you need to either use RPC or try to call a function that triggers "Montage play" inside repNotify function <- matter of choice and discussion long story short look for RPC and use it to play montages.
